content/en/tracing/troubleshooting/_index.md

```diff
@@ -106,64 +254,92 @@ Second primary tags are additional tags that you can use to group and aggregate
+
+{{% /collapse-content %}}
+
+## Resource usage
+
+This section contains information on troubleshooting performance issues related to resource utilization.
+
+{{% collapse-content title="Out of memory errors" level="h4" %}}
+
+Read about detecting trace collection CPU usage and about calculating adequate resource limits for the Agent in [Agent Resource Usage][10].
+
+{{% /collapse-content %}}
 
-## Troubleshooting data requested by Datadog Support
+{{% collapse-content title="Rate limit or max event error messages" level="h4" %}}
 
-When you open a [support ticket][1], our support team may ask for some combination of the following types of information:
+Within Datadog Agent logs, if you see error messages about rate limits or max events per second, you can change these limits by following [these instructions][9]. If you have questions, before you change the limits, consult with the Datadog [support team][1].
+
+{{% /collapse-content %}}
+
+## Security
+
+This section covers approaches for addressing security concerns in APM, including protecting sensitive data and managing traffic.
+
+{{% collapse-content title="Modifying, discarding, or obfuscating spans" level="h4" %}}
+
+There are several configuration options available to scrub sensitive data or discard traces corresponding to health checks or other unwanted traffic that can be configured within the Datadog Agent, or in some languages the Tracing Client. For details on the options available, see [Security and Agent Customization][11]. While this offers representative examples, if you require assistance applying these options to your environment, reach out to [Datadog Support][1].
 
-1. **How are you confirming the issue? Provide links to a trace (preferably) or screenshots, for example, and tell support what you expect to see.**
+{{% /collapse-content %}}
 
-    This allows Support to confirm errors and attempt to reproduce your issues within Datadog's testing environments.
+## Debugging and logging
 
-2. **[Tracer startup logs](#confirm-apm-setup-and-agent-status)**
+This section explains how to use debug and startup logs to identify and resolve issues with your Datadog tracer.
 
-    Startup logs are a great way to spot misconfiguration of the tracer, or the inability for the tracer to communicate with the Datadog Agent. By comparing the configuration that the tracer sees to the one set within the application or container, Support can identify areas where a setting is not being properly applied.
+{{% collapse-content title="Debug logs" level="h4" %}}
 
-3. **[Tracer debug logs](#tracer-debug-logs)**
+To capture full details on the Datadog tracer, enable debug mode on your tracer by using the `DD_TRACE_DEBUG` environment variable. You might enable it for your own investigation or because Datadog support recommended it for triage purposes. However, don't leave debug mode always enabled because of the logging overhead it introduces.
```

Comment:
```suggestion
To capture full details on the Datadog tracer, enable debug mode on your tracer by using the `DD_TRACE_DEBUG` environment variable. You might enable it for your own investigation or if Datadog support has recommended it for triage purposes. However, be sure to disable debug logging when you are finished testing to avoid the logging overhead it introduces.
```